### ISSUE TITLE
Failed to start with `xonsh`

### ISSUE THREAD

**Issue by doronz88** [microsoft/inshellisense#208]
**Describe the bug**
Trying to start inshellisense with `xonsh` as my default shell provides the following exception and doesn't do a thing:

```
/opt/homebrew/Cellar/python@3.11/3.11.7_1/Frameworks/Python.framework/Versions/3.11/Resources/Python.app/Contents/MacOS/Python: can't open file '/Users/user/xonsh': [Errno 2] No such file or directory
```

**To Reproduce**
- Execute: `is`

**Expected behavior**
Brighten my day :)

**Environment**
 - OS: macOS 14.4
 - Output of `is --version`: `0.0.1-rc.9`
 - Nodejs Version: `v20.11.1`
 - Shell: `xonsh`
 - Shell Version: `0.15.1`

**Comment by cpendery** [comment]
@doronz88 how did you install xonsh? Is it available on your path? Right now, I check for xonsh in your python modules since that was the recommended way to do a cross platform install last I checked the docs

**Comment by doronz88** [comment]
I installed it using pipx (which is generally the  recommended for CLI utils)

**Comment by doronz88** [comment]
The issue persists with version `0.0.1-rc.10`

**Comment by cpendery** [comment]
@doronz88, I need more information to replicate your failing xonsh startup. Can you run with the `-V` flag and provide the logs here?

**Comment by doronz88** [comment]
Do you mean `is -V`? It gives the exact same output. It performs a "clear-screen" and just prints:
```
/opt/homebrew/Cellar/python@3.11/3.11.8/Frameworks/Python.framework/Versions/3.11/Resources/Python.app/Contents/MacOS/Python: can't open file '/Users/user/dev/downloader/xonsh': [Errno 2] No such file or directory
```

**Comment by cpendery** [comment]
> Do you mean `is -V`? It gives the exact same output. It performs a "clear-screen" and just prints:
> 
> ```
> /opt/homebrew/Cellar/python@3.11/3.11.8/Frameworks/Python.framework/Versions/3.11/Resources/Python.app/Contents/MacOS/Python: can't open file '/Users/user/dev/downloader/xonsh': [Errno 2] No such file or directory
> ```

It should create a log file under ~/.inshellisense/inshellisense.log (sorry for not mentioning that earlier). Could you provide that file's details here?

**Comment by doronz88** [comment]
```
{"msg":"writing data","data":"\u001b[2J\u001b[3J\u001b[H"}
{"msg":"parsing data","data":"/opt/homebrew/Cellar/python@3.11/3.11.8/Frameworks/Python.framework/Versions/3.11/Resources/Python.app/Contents/MacOS/Python: can't open file '/Users/user/xonsh': [Errno 2] No such file or directory\r\n","bytes":{"0":47,"1":111,"2":112,"3":116,"4":47,"5":104,"6":111,"7":109,"8":101,"9":98,"10":114,"11":101,"12":119,"13":47,"14":67,"15":101,"16":108,"17":108,"18":97,"19":114,"20":47,"21":112,"22":121,"23":116,"24":104,"25":111,"26":110,"27":64,"28":51,"29":46,"30":49,"31":49,"32":47,"33":51,"34":46,"35":49,"36":49,"37":46,"38":56,"39":47,"40":70,"41":114,"42":97,"43":109,"44":101,"45":119,"46":111,"47":114,"48":107,"49":115,"50":47,"51":80,"52":121,"53":116,"54":104,"55":111,"56":110,"57":46,"58":102,"59":114,"60":97,"61":109,"62":101,"63":119,"64":111,"65":114,"66":107,"67":47,"68":86,"69":101,"70":114,"71":115,"72":105,"73":111,"74":110,"75":115,"76":47,"77":51,"78":46,"79":49,"80":49,"81":47,"82":82,"83":101,"84":115,"85":111,"86":117,"87":114,"88":99,"89":101,"90":115,"91":47,"92":80,"93":121,"94":116,"95":104,"96":111,"97":110,"98":46,"99":97,"100":112,"101":112,"102":47,"103":67,"104":111,"105":110,"106":116,"107":101,"108":110,"109":116,"110":115,"111":47,"112":77,"113":97,"114":99,"115":79,"116":83,"117":47,"118":80,"119":121,"120":116,"121":104,"122":111,"123":110,"124":58,"125":32,"126":99,"127":97,"128":110,"129":39,"130":116,"131":32,"132":111,"133":112,"134":101,"135":110,"136":32,"137":102,"138":105,"139":108,"140":101,"141":32,"142":39,"143":47,"144":85,"145":115,"146":101,"147":114,"148":115,"149":47,"150":117,"151":115,"152":101,"153":114,"154":47,"155":120,"156":111,"157":110,"158":115,"159":104,"160":39,"161":58,"162":32,"163":91,"164":69,"165":114,"166":114,"167":110,"168":111,"169":32,"170":50,"171":93,"172":32,"173":78,"174":111,"175":32,"176":115,"177":117,"178":99,"179":104,"180":32,"181":102,"182":105,"183":108,"184":101,"185":32,"186":111,"187":114,"188":32,"189":100,"190":105,"191":114,"192":101,"193":99,"194":116,"195":111,"196":114,"197":121,"198":13,"199":10}}
{"msg":"writing data","data":"/opt/homebrew/Cellar/python@3.11/3.11.8/Frameworks/Python.framework/Versions/3.11/Resources/Python.app/Contents/MacOS/Python: can't open file '/Users/user/xonsh': [Errno 2] No such file or directory\r\n"}
```

**Comment by cpendery** [comment]
@doronz88, can you please confirm that it's working now for you with `0.0.1-rc.11`. It just released

**Comment by doronz88** [comment]
We're almost there! 
No exception, but completions are irrelevant 
<img width="724" alt="image" src="https://github.com/microsoft/inshellisense/assets/19665147/cda48c93-5a84-4501-92f5-43d4f09dbf5b">

**Comment by cpendery** [comment]
Can you provide the logs again? It looks like the suggestion text is getting picked up as active input

**Comment by doronz88** [comment]
Sure!

```
{"msg":"writing data","data":"\u001b[2J\u001b[3J\u001b[H"}
{"msg":"parsing data","data":"\r","bytes":{"0":13}}
{"msg":"writing data","data":"\r"}
{"msg":"parsing data","data":"\u001b]0;doron@MacBook-Pro: ~ | xonsh\u0007","bytes":{"0":27,"1":93,"2":48,"3":59,"4":100,"5":111,"6":114,"7":111,"8":110,"9":64,"10":77,"11":97,"12":99,"13":66,"14":111,"15":111,"16":107,"17":45,"18":80,"19":114,"20":111,"21":58,"22":32,"23":126,"24":32,"25":124,"26":32,"27":120,"28":111,"29":110,"30":115,"31":104,"32":7}}
{"msg":"writing data","data":"\u001b]0;doron@MacBook-Pro: ~ | xonsh\u0007"}
{"msg":"parsing data","data":"\u001b[6n","bytes":{"0":27,"1":91,"2":54,"3":110}}
{"msg":"writing data","data":"\u001b[6n"}
{"msg":"reading data","data":"\u001b[1;1R","bytes":{"0":27,"1":91,"2":49,"3":59,"4":49,"5":82}}
{"msg":"parsing data","data":"\u001b[?2004h\u001b[?1l\u001b[?25l\u001b[0m\u001b[?7l\u001b[0m\u001b[J\u001b]6973;PS\u0007\u001b]6973;CWD;/Users/doron\u0007\u001b[0;32;1mdoron@MacBook-Pro\u001b[0;34;1m ~\u001b[0m \u001b[0;34;1m@\u001b[21D\u001b[22C\u001b[?7h\u001b[0m\u001b[?12l\u001b[?25h","bytes":{"0":27,"1":91,"2":63,"3":50,"4":48,"5":48,"6":52,"7":104,"8":27,"9":91,"10":63,"11":49,"12":108,"13":27,"14":91,"15":63,"16":50,"17":53,"18":108,"19":27,"20":91,"21":48,"22":109,"23":27,"24":91,"25":63,"26":55,"27":108,"28":27,"29":91,"30":48,"31":109,"32":27,"33":91,"34":74,"35":27,"36":93,"37":54,"38":57,"39":55,"40":51,"41":59,"42":80,"43":83,"44":7,"45":27,"46":93,"47":54,"48":57,"49":55,"50":51,"51":59,"52":67,"53":87,"54":68,"55":59,"56":47,"57":85,"58":115,"59":101,"60":114,"61":115,"62":47,"63":100,"64":111,"65":114,"66":111,"67":110,"68":7,"69":27,"70":91,"71":48,"72":59,"73":51,"74":50,"75":59,"76":49,"77":109,"78":100,"79":111,"80":114,"81":111,"82":110,"83":64,"84":77,"85":97,"86":99,"87":66,"88":111,"89":111,"90":107,"91":45,"92":80,"93":114,"94":111,"95":27,"96":91,"97":48,"98":59,"99":51,"100":52,"101":59,"102":49,"103":109,"104":32,"105":126,"106":27,"107":91,"108":48,"109":109,"110":32,"111":27,"112":91,"113":48,"114":59,"115":51,"116":52,"117":59,"118":49,"119":109,"120":64,"121":27,"122":91,"123":50,"124":49,"125":68,"126":27,"127":91,"128":50,"129":50,"130":67,"131":27,"132":91,"133":63,"134":55,"135":104,"136":27,"137":91,"138":48,"139":109,"140":27,"141":91,"142":63,"143":49,"144":50,"145":108,"146":27,"147":91,"148":63,"149":50,"150":53,"151":104}}
{"msg":"writing data","data":"\u001b[?2004h\u001b[?1l\u001b[?25l\u001b[0m\u001b[?7l\u001b[0m\u001b[J\u001b]6973;PS\u0007\u001b]6973;CWD;/Users/doron\u0007\u001b[0;32;1mdoron@MacBook-Pro\u001b[0;34;1m ~\u001b[0m \u001b[0;34;1m@\u001b[21D\u001b[22C\u001b[?7h\u001b[0m\u001b[?12l\u001b[?25h"}
{"msg":"parsing data","data":"\u001b[?25l\u001b[?7l\u001b[22D\u001b]6973;PS\u0007\u001b]6973;CWD;/Users/doron\u0007\u001b[0;32;1mdoron@MacBook-Pro\u001b[0;34;1m ~\u001b[0m \u001b[0;34;1m@\u001b[0m\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\u001b[0m \b\u001b[46A\u001b[22C\u001b[?7h\u001b[0m\u001b[?12l\u001b[?25h","bytes":{"0":27,"1":91,"2":63,"3":50,"4":53,"5":108,"6":27,"7":91,"8":63,"9":55,"10":108,"11":27,"12":91,"13":50,"14":50,"15":68,"16":27,"17":93,"18":54,"19":57,"20":55,"21":51,"22":59,"23":80,"24":83,"25":7,"26":27,"27":93,"28":54,"29":57,"30":55,"31":51,"32":59,"33":67,"34":87,"35":68,"36":59,"37":47,"38":85,"39":115,"40":101,"41":114,"42":115,"43":47,"44":100,"45":111,"46":114,"47":111,"48":110,"49":7,"50":27,"51":91,"52":48,"53":59,"54":51,"55":50,"56":59,"57":49,"58":109,"59":100,"60":111,"61":114,"62":111,"63":110,"64":64,"65":77,"66":97,"67":99,"68":66,"69":111,"70":111,"71":107,"72":45,"73":80,"74":114,"75":111,"76":27,"77":91,"78":48,"79":59,"80":51,"81":52,"82":59,"83":49,"84":109,"85":32,"86":126,"87":27,"88":91,"89":48,"90":109,"91":32,"92":27,"93":91,"94":48,"95":59,"96":51,"97":52,"98":59,"99":49,"100":109,"101":64,"102":27,"103":91,"104":48,"105":109,"106":13,"107":13,"108":10,"109":13,"110":13,"111":10,"112":13,"113":13,"114":10,"115":13,"116":13,"117":10,"118":13,"119":13,"120":10,"121":13,"122":13,"123":10,"124":13,"125":13,"126":10,"127":13,"128":13,"129":10,"130":13,"131":13,"132":10,"133":13,"134":13,"135":10,"136":13,"137":13,"138":10,"139":13,"140":13,"141":10,"142":13,"143":13,"144":10,"145":13,"146":13,"147":10,"148":13,"149":13,"150":10,"151":13,"152":13,"153":10,"154":13,"155":13,"156":10,"157":13,"158":13,"159":10,"160":13,"161":13,"162":10,"163":13,"164":13,"165":10,"166":13,"167":13,"168":10,"169":13,"170":13,"171":10,"172":13,"173":13,"174":10,"175":13,"176":13,"177":10,"178":13,"179":13,"180":10,"181":13,"182":13,"183":10,"184":13,"185":13,"186":10,"187":13,"188":13,"189":10,"190":13,"191":13,"192":10,"193":13,"194":13,"195":10,"196":13,"197":13,"198":10,"199":13,"200":13,"201":10,"202":13,"203":13,"204":10,"205":13,"206":13,"207":10,"208":13,"209":13,"210":10,"211":13,"212":13,"213":10,"214":13,"215":13,"216":10,"217":13,"218":13,"219":10,"220":13,"221":13,"222":10,"223":13,"224":13,"225":10,"226":13,"227":13,"228":10,"229":13,"230":13,"231":10,"232":13,"233":13,"234":10,"235":13,"236":13,"237":10,"238":13,"239":13,"240":10,"241":13,"242":13,"243":10,"244":27,"245":91,"246":48,"247":109,"248":32,"249":8,"250":27,"251":91,"252":52,"253":54,"254":65,"255":27,"256":91,"257":50,"258":50,"259":67,"260":27,"261":91,"262":63,"263":55,"264":104,"265":27,"266":91,"267":48,"268":109,"269":27,"270":91,"271":63,"272":49,"273":50,"274":108,"275":27,"276":91,"277":63,"278":50,"279":53,"280":104}}
{"msg":"writing data","data":"\u001b[?25l\u001b[?7l\u001b[22D\u001b]6973;PS\u0007\u001b]6973;CWD;/Users/doron\u0007\u001b[0;32;1mdoron@MacBook-Pro\u001b[0;34;1m ~\u001b[0m \u001b[0;34;1m@\u001b[0m\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\r\r\n\u001b[0m \b\u001b[46A\u001b[22C\u001b[?7h\u001b[0m\u001b[?12l\u001b[?25h"}
{"msg":"reading data","data":"n","bytes":{"0":110}}
{"msg":"parsing data","data":"\u001b[?25l\u001b[?7l\u001b]6973;PE\u0007\u001b[0;38;5;108mn\u001b[0;90minstall\u001b[7D\u001b[?7h\u001b[0m\u001b[?12l\u001b[?25h","bytes":{"0":27,"1":91,"2":63,"3":50,"4":53,"5":108,"6":27,"7":91,"8":63,"9":55,"10":108,"11":27,"12":93,"13":54,"14":57,"15":55,"16":51,"17":59,"18":80,"19":69,"20":7,"21":27,"22":91,"23":48,"24":59,"25":51,"26":56,"27":59,"28":53,"29":59,"30":49,"31":48,"32":56,"33":109,"34":110,"35":27,"36":91,"37":48,"38":59,"39":57,"40":48,"41":109,"42":105,"43":110,"44":115,"45":116,"46":97,"47":108,"48":108,"49":27,"50":91,"51":55,"52":68,"53":27,"54":91,"55":63,"56":55,"57":104,"58":27,"59":91,"60":48,"61":109,"62":27,"63":91,"64":63,"65":49,"66":50,"67":108,"68":27,"69":91,"70":63,"71":50,"72":53,"73":104}}
{"msg":"cmd manager state","promptStartMarker":0,"hasOutput":false,"cursorTerminated":true,"promptEndMarker":0,"promptEndX":22,"promptText":"doron@MacBook-Pro ~ @","persistentOutput":false,"suggestionsText":"install","commandText":"n "}
{"msg":"writing data","data":"\u001b[?25l\u001b[?7l\u001b]6973;PE\u0007\u001b[0;38;5;108mn\u001b[0;90minstall\u001b[7D\u001b[?7h\u001b[0m\u001b[?12l\u001b[?25h"}
{"msg":"shell command failed","e":"spawn n ENOENT"}
{"msg":"writing data","data":"\u001b[?25l\u001b[s\u001b[E\u001b[E\u001b[E\u001b[E\u001b[E\u001b[E\u001b[E\u001b[?25l\u001b[23C\u001b[6A\u001b[24G┌─────────────────────────xterm.js: Parsing error:  {
  position: 663,
  code: 127,
  currentState: 0,
  collect: 0,
  params: s {
    maxLength: 32,
    maxSubParamsLength: 32,
    params: Int32Array(32) [
      0, 38, 5, 187, 1, 0, 0, 0, 0,
      0,  0, 0,   0, 0, 0, 0, 0, 0,
      0,  0, 0,   0, 0, 0, 0, 0, 0,
      0,  0, 0,   0, 0
    ],
    length: 1,
    _subParams: Int32Array(32) [
      0, 0, 0, 0, 0, 0, 0, 0, 0,
      0, 0, 0, 0, 0, 0, 0, 0, 0,
      0, 0, 0, 0, 0, 0, 0, 0, 0,
      0, 0, 0, 0, 0
    ],
    _subParamsLength: 0,
    _subParamsIdx: Uint16Array(32) [
      0, 0, 0, 0, 0, 0, 0, 0, 0,
      0, 0, 0, 0, 0, 0, 0, 0, 0,
      0, 0, 0, 0, 0, 0, 0, 0, 0,
      0, 0, 0, 0, 0
    ],
    _rejectDigits: false,
    _rejectSubDigits: false,
    _digitIsSub: false
  },
  abort: false
}
─────────────┐\u001b[24G\u001b[1B│\u001b[48;2;125;86;244m📦 install                            \u001b[49m│\u001b[24G\u001b[1B│📦 rm                                 │\u001b[24G\u001b[1B│📦 prune                              │\u001b[24G\u001b[1B│📦 doctor                             │\u001b[24G\u001b[1B│📦 uninstall                          │\u001b[24G\u001b[1B└──────────────────────────────────────┘\u001b[24G\u001b[6A\u001b[6B\u001b[6A\u001b[64G┌────────────────────────────┐\u001b[64G\u001b[1B│Install a Node.js version   │\u001b[64G\u001b[1B└────────────────────────────┘\u001b[64G\u001b[2A\u001b[6B\u001b[?25h\u001b[u\u001b[?25h"}
{"msg":"reading data","data":"","bytes":{"0":127}}
{"msg":"parsing data","data":"\u001b[?25l\u001b[?7l\u001b[2D\u001b[0m\u001b[K\u001b[C\u001b[?7h\u001b[0m\u001b[?12l\u001b[?25h","bytes":{"0":27,"1":91,"2":63,"3":50,"4":53,"5":108,"6":27,"7":91,"8":63,"9":55,"10":108,"11":27,"12":91,"13":50,"14":68,"15":27,"16":91,"17":48,"18":109,"19":27,"20":91,"21":75,"22":27,"23":91,"24":67,"25":27,"26":91,"27":63,"28":55,"29":104,"30":27,"31":91,"32":48,"33":109,"34":27,"35":91,"36":63,"37":49,"38":50,"39":108,"40":27,"41":91,"42":63,"43":50,"44":53,"45":104}}
{"msg":"writing data","data":"\u001b[?25l\u001b[s\u001b[E\u001b[2K\u001b[E\u001b[2K\u001b[E\u001b[2K\u001b[E\u001b[2K\u001b[E\u001b[2K\u001b[E\u001b[2K\u001b[E\u001b[2K\u001b[u\u001b[?25h"}
{"msg":"writing data","data":"\u001b[?25l\u001b[s\u001b[E\u001b[2K\u001b[E\u001b[2K\u001b[E\u001b[2K\u001b[E\u001b[2K\u001b[E\u001b[2K\u001b[E\u001b[2K\u001b[E\u001b[2K\u001b[E\u001b[2K\u001b[u\u001b[?25h\u001b[?25l\u001b[?7l\u001b[2D\u001b[0m\u001b[K\u001b[C\u001b[?7h\u001b[0m\u001b[?12l\u001b[?25h"}
{"msg":"cmd manager state","promptStartMarker":0,"hasOutput":false,"cursorTerminated":true,"proxterm.js: Parsing error:  {
  position: 165,
  code: 127,
  currentState: 0,
  collect: 0,
  params: s {
    maxLength: 32,
    maxSubParamsLength: 32,
    params: Int32Array(32) [
      0, 38, 5, 187, 1, 0, 0, 0, 0,
      0,  0, 0,   0, 0, 0, 0, 0, 0,
      0,  0, 0,   0, 0, 0, 0, 0, 0,
      0,  0, 0,   0, 0
    ],
    length: 1,
    _subParams: Int32Array(32) [
      0, 0, 0, 0, 0, 0, 0, 0, 0,
      0, 0, 0, 0, 0, 0, 0, 0, 0,
      0, 0, 0, 0, 0, 0, 0, 0, 0,
      0, 0, 0, 0, 0
    ],
    _subParamsLength: 0,
    _subParamsIdx: Uint16Array(32) [
      0, 0, 0, 0, 0, 0, 0, 0, 0,
      0, 0, 0, 0, 0, 0, 0, 0, 0,
      0, 0, 0, 0, 0, 0, 0, 0, 0,
      0, 0, 0, 0, 0
    ],
    _rejectDigits: false,
    _rejectSubDigits: false,
    _digitIsSub: false
  },
  abort: false
}
mptEndMarker":0,"promptEndX":22,"promptText":"doron@MacBook-Pro ~ @","persistentOutput":false,"suggestionsText":"","commandText":" "}
{"msg":"reading data","data":"","bytes":{"0":127}}
{"msg":"parsing data","data":"\u0007","bytes":{"0":7}}
{"msg":"cmd manager state","promptStartMarker":0,"hasOutput":false,"cursorTerminated":true,"promptEndMarker":0,"promptEndX":22,"promptText":"doron@MacBook-Pro ~ @","persistentOutput":false,"suggestionsText":"","commandText":" "}
{"msg":"writing data","data":"\u0007"}
{"msg":"parsing data","data":"\u001b[?25l\u001b[?7l\u001b[?7h\u001b[0m\u001b[?12l\u001b[?25h","bytes":{"0":27,"1":91,"2":63,"3":50,"4":53,"5":108,"6":27,"7":91,"8":63,"9":55,"10":108,"11":27,"12":91,"13":63,"14":55,"15":104,"16":27,"17":91,"18":48,"19":109,"20":27,"21":91,"22":63,"23":49,"24":50,"25":108,"26":27,"27":91,"28":63,"29":50,"30":53,"31":104}}
{"msg":"writing data","data":"\u001b[?25l\u001b[?7l\u001b[?7h\u001b[0m\u001b[?12l\u001b[?25h"}
{"msg":"cmd manager state","promptStartMarker":0,"hasOutput":false,"cursorTerminated":true,"promptEndMarker":0,"promptEndX":22,"promptText":"doron@MacBook-Pro ~ @","persistentOutput":false,"suggestionsText":"","commandText":" "}
{"msg":"reading data","data":"c","bytes":{"0":99}}
{"msg":"parsing data","data":"\u001b[?25l\u001b[?7l\u001b]6973;PE\u0007\u001b[0;38;5;108mc\u001b[0;90mharm .\u001b[6D\u001b[?7h\u001b[0m\u001b[?12l\u001b[?25h","bytes":{"0":27,"1":91,"2":63,"3":50,"4":53,"5":108,"6":27,"7":91,"8":63,"9":55,"10":108,"11":27,"12":93,"13":54,"14":57,"15":55,"16":51,"17":59,"18":80,"19":69,"20":7,"21":27,"22":91,"23":48,"24":59,"25":51,"26":56,"27":59,"28":53,"29":59,"30":49,"31":48,"32":56,"33":109,"34":99,"35":27,"36":91,"37":48,"38":59,"39":57,"40":48,"41":109,"42":104,"43":97,"44":114,"45":109,"46":32,"47":46,"48":27,"49":91,"50":54,"51":68,"52":27,"53":91,"54":63,"55":55,"56":104,"57":27,"58":91,"59":48,"60":109,"61":27,"62":91,"63":63,"64":49,"65":50,"66":108,"67":27,"68":91,"69":63,"70":50,"71":53,"72":104}}
{"msg":"cmd manager state","promptStartMarker":0,"hasOutput":false,"cursorTerminated":true,"promptEndMarker":0,"promptEndX":22,"promptText":"doron@MacBook-Pro ~ @","persistentOutput":false,"suggestionsText":"harm .","commandText":"c "}
{"msg":"writing data","data":"\u001b[?25l\u001b[?7l\u001b]6973;PE\u0007\u001b[0;38;5;108mc\u001b[0;90mharm .\u001b[6D\u001b[?7h\u001b[0m\u001b[?12l\u001b[?25h"}
{"msg":"reading data","data":"a","bytes":{"0":97}}
{"msg":"parsing data","data":"\u001b[?25l\u001b[?7l\u001b[0;38;5;108ma\u001b[0;90mt ~/\u001b[Cssh/id_rsa.pub\u001b[19D\u001b[?7h\u001b[0m\u001b[?12l\u001b[?25h","bytes":{"0":27,"1":91,"2":63,"3":50,"4":53,"5":108,"6":27,"7":91,"8":63,"9":55,"10":108,"11":27,"12":91,"13":48,"14":59,"15":51,"16":56,"17":59,"18":53,"19":59,"20":49,"21":48,"22":56,"23":109,"24":97,"25":27,"26":91,"27":48,"28":59,"29":57,"30":48,"31":109,"32":116,"33":32,"34":126,"35":47,"36":27,"37":91,"38":67,"39":115,"40":115,"41":104,"42":47,"43":105,"44":100,"45":95,"46":114,"47":115,"48":97,"49":46,"50":112,"51":117,"52":98,"53":27,"54":91,"55":49,"56":57,"57":68,"58":27,"59":91,"60":63,"61":55,"62":104,"63":27,"64":91,"65":48,"66":109,"67":27,"68":91,"69":63,"70":49,"71":50,"72":108,"73":27,"74":91,"75":63,"76":50,"77":53,"78":104}}
{"msg":"cmd manager state","promptStartMarker":0,"hasOutput":false,"cursorTerminated":true,"promptEndMarker":0,"promptEndX":22,"promptText":"doron@MacBook-Pro ~ @","persistentOutput":false,"suggestionsText":"t ~/.ssh/id_rsa.pub","commandText":"ca "}
{"msg":"writing data","data":"\u001b[?25l\u001b[?7l\u001b[0;38;5;108ma\u001b[0;90mt ~/\u001b[Cssh/id_rsa.pub\u001b[19D\u001b[?7h\u001b[0m\u001b[?12l\u001b[?25h"}
{"msg":"reading data","data":"t","bytes":{"0":116}}
{"msg":"parsing data","data":"\u001b[?25l\u001b[?7l\u001b[2D\u001b]6973;PE\u0007\u001b[0;38;5;208;1mcat\u001b[?7h\u001b[0m\u001b[?12l\u001b[?25h","bytes":{"0":27,"1":91,"2":63,"3":50,"4":53,"5":108,"6":27,"7":91,"8":63,"9":55,"10":108,"11":27,"12":91,"13":50,"14":68,"15":27,"16":93,"17":54,"18":57,"19":55,"20":51,"21":59,"22":80,"23":69,"24":7,"25":27,"26":91,"27":48,"28":59,"29":51,"30":56,"31":59,"32":53,"33":59,"34":50,"35":48,"36":56,"37":59,"38":49,"39":109,"40":99,"41":97,"42":116,"43":27,"44":91,"45":63,"46":55,"47":104,"48":27,"49":91,"50":48,"51":109,"52":27,"53":91,"54":63,"55":49,"56":50,"57":108,"58":27,"59":91,"60":63,"61":50,"62":53,"63":104}}
{"msg":"writing data","data":"\u001b[?25l\u001b[?7l\u001b[2D\u001b]6973;PE\u0007\u001b[0;38;5;208;1mcat\u001b[?7h\u001b[0m\u001b[?12l\u001b[?25h"}
{"msg":"cmd manager state","promptStartMarker":0,"hasOutput":false,"cursorTerminated":false,"promptEndMarker":0,"promptEndX":22,"promptText":"doron@MacBook-Pro ~ @ cat ~/.ssh/id_rsa.pub","persistentOutput":false,"suggestionsText":"","commandText":""}
{"msg":"reading data","data":" ","bytes":{"0":32}}
{"msg":"parsing data","data":"\u001b[?25l\u001b[?7l\u001b[0;38;5;187m \u001b[?7h\u001b[0m\u001b[?12l\u001b[?25h","bytes":{"0":27,"1":91,"2":63,"3":50,"4":53,"5":108,"6":27,"7":91,"8":63,"9":55,"10":108,"11":27,"12":91,"13":48,"14":59,"15":51,"16":56,"17":59,"18":53,"19":59,"20":49,"21":56,"22":55,"23":109,"24":32,"25":27,"26":91,"27":63,"28":55,"29":104,"30":27,"31":91,"32":48,"33":109,"34":27,"35":91,"36":63,"37":49,"38":50,"39":108,"40":27,"41":91,"42":63,"43":50,"44":53,"45":104}}
{"msg":"writing data","data":"\u001b[?25l\u001b[?7l\u001b[0;38;5;187m \u001b[?7h\u001b[0m\u001b[?12l\u001b[?25h"}
{"msg":"cmd manager state","promptStartMarker":0,"hasOutput":false,"cursorTerminated":false,"promptEndMarker":0,"promptEndX":22,"promptText":"doron@MacBook-Pro ~ @ cat ~/.ssh/id_rsa.pub","persistentOutput":false,"suggestionsText":"","commandText":""}
{"msg":"reading data","data":"\u001b[200~","bytes":{"0":27,"1":91,"2":50,"3":48,"4":48,"5":126}}
{"msg":"reading data","data":"/","bytes":{"0":47}}
{"msg":"reading data","data":"i","bytes":{"0":105}}
{"msg":"reading data","data":".","bytes":{"0":46}}
{"msg":"reading data","data":"n","bytes":{"0":110}}
{"msg":"reading data","data":"s","bytes":{"0":115}}
{"msg":"reading data","data":"h","bytes":{"0":104}}
{"msg":"reading data","data":"e","bytes":{"0":101}}
{"msg":"reading data","data":"i","bytes":{"0":105}}
{"msg":"reading data","data":"l","bytes":{"0":108}}
{"msg":"reading data","data":"l","bytes":{"0":108}}
{"msg":"reading data","data":"s","bytes":{"0":115}}
{"msg":"reading data","data":"e","bytes":{"0":101}}
{"msg":"reading data","data":"~","bytes":{"0":126}}
{"msg":"reading data","data":"e","bytes":{"0":101}}
{"msg":"reading data","data":"n","bytes":{"0":110}}
{"msg":"reading data","data":"s","bytes":{"0":115}}
{"msg":"reading data","data":"/","bytes":{"0":47}}
{"msg":"reading data","data":"n","bytes":{"0":110}}
{"msg":"reading data","data":"s","bytes":{"0":115}}
{"msg":"reading data","data":"i","bytes":{"0":105}}
{"msg":"reading data","data":"l","bytes":{"0":108}}
{"msg":"reading data","data":"l","bytes":{"0":108}}
{"msg":"reading data","data":"e","bytes":{"0":101}}
{"msg":"reading data","data":"i","bytes":{"0":105}}
{"msg":"reading data","data":"e","bytes":{"0":101}}
{"msg":"reading data","data":"n","bytes":{"0":110}}
{"msg":"reading data","data":"h","bytes":{"0":104}}
{"msg":"reading data","data":"s","bytes":{"0":115}}
{"msg":"reading data","data":"s","bytes":{"0":115}}
{"msg":"reading data","data":"e","bytes":{"0":101}}
{"msg":"reading data","data":".","bytes":{"0":46}}
{"msg":"reading data","data":"g","bytes":{"0":103}}
{"msg":"reading data","data":"l","bytes":{"0":108}}
{"msg":"reading data","data":"o","bytes":{"0":111}}
{"msg":"reading data","data":"\u001b[201~","bytes":{"0":27,"1":91,"2":50,"3":48,"4":49,"5":126}}
{"msg":"parsing data","data":"\u001b[?25l\u001b[?7l\u001b[0m~/.inshellisense/inshellisense.log\u001b[?7h\u001b[0m\u001b[?12l\u001b[?25h","bytes":{"0":27,"1":91,"2":63,"3":50,"4":53,"5":108,"6":27,"7":91,"8":63,"9":55,"10":108,"11":27,"12":91,"13":48,"14":109,"15":126,"16":47,"17":46,"18":105,"19":110,"20":115,"21":104,"22":101,"23":108,"24":108,"25":105,"26":115,"27":101,"28":110,"29":115,"30":101,"31":47,"32":105,"33":110,"34":115,"35":104,"36":101,"37":108,"38":108,"39":105,"40":115,"41":101,"42":110,"43":115,"44":101,"45":46,"46":108,"47":111,"48":103,"49":27,"50":91,"51":63,"52":55,"53":104,"54":27,"55":91,"56":48,"57":109,"58":27,"59":91,"60":63,"61":49,"62":50,"63":108,"64":27,"65":91,"66":63,"67":50,"68":53,"69":104}}
{"msg":"cmd manager state","promptStartMarker":0,"hasOutput":false,"cursorTerminated":true,"promptEndMarker":0,"promptEndX":22,"promptText":"doron@MacBook-Pro ~ @ cat ~/.ssh/id_rsa.pub","persistentOutput":false,"suggestionsText":"","commandText":"inshellisense.log"}
{"msg":"writing data","data":"\u001b[?25l\u001b[?7l\u001b[0m~/.inshellisense/inshellisense.log\u001b[?7h\u001b[0m\u001b[?12l\u001b[?25h"}
{"msg":"reading data","data":"\r","bytes":{"0":13}}
```

**Comment by doronz88** [comment]
Waiting for your next release 🥹

**Comment by cpendery** [comment]
@doronz88 just released `0.0.1-rc.12`!

**Comment by doronz88** [comment]
Can verify this works like a charm!
Now the expected behavior which is to brighten my day is fully accomplished!

Thanks! 🙏🔥🚀
